lib/config/app_config.dart

```diff
@@ -24,6 +27,8 @@ abstract class AppConfig {
   static String _privacyUrl = 'https://twake.app/en/privacy/';
 
   static String get privacyUrl => _privacyUrl;
+  static const String registationUrl =
```

Comment:
Please do it nullable. Hide this button when it is not configured.

Reply:
it can't nullable because default value is "". I have hided the button if registationUrl is empty

Reply:
what happen if the configuration dont have this field?

Reply:
It will be "". which is empty string
<img width="792" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/8d87f550-8288-45e3-97f3-3335ea65ca49">

Reply:
and i have handled it in here
<img width="499" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/3597d84e-b967-4b56-a71f-c99fc99cb43a">

Reply:
`registrationUrl`